mautic/mautic
# "Don't overwrite value if already exists" option is not working In Import Contacts.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [N]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [N]
| Automated tests included?              | [Y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
"Don't overwrite value if already exists" option is not working In Import



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create or use a sample CV from this ticket.
3. Try uploading that csv.
4. Then try uploading another time by selecting Don't overwrite the value if it already exists.
5. When flag is ON we should not update the value of field is already set(Non Empty) . If value is empty then it should get updated with new value.
